scp_on_cloud_node.py

```diff
@@ -0,0 +1,25 @@
+from reefperf.cloud_driver import LCCloudScaleDriver
+import paramiko
+from pprint import pprint
+import libcloud
+import paramiko
+from io import StringIO
+
+if __name__ == '__main__':
+    credentials = {
+        'api_token': 'token'
+    }
+    cloud_node_cfg = {
+        'node_name': 'node1',
+        'size': 'flex-2',
+        'image': 'ubuntu-16.04',
+        'connection': {
+            'connection_class': 'paramiko'
+        }
+    }
+    driver = LCCloudScaleDriver(credentials)
+    node = driver.create_node(cloud_node_cfg)
+    stdout_ = node.connection.exec_command('ls /')
+    for line in stdout_.readlines():
+        print(line)
+    node.destroy()
```

Comment:
if there will be any failure between creation and destruction of the node, exception will be raised and node will not get destroyed. That is why context managers were built - please use one here.